include/dxc/dxcpix.h

```diff
@@ -47,12 +47,23 @@ struct __declspec(uuid("9ba0d9d3-457b-426f-8019-9f3849982aa2")) IDxcPixArrayType
 };
 
 struct __declspec(uuid("6c707d08-7995-4a84-bae5-e6d8291f3b78"))
+    IDxcPixStructField0 : public IUnknown {
```

Comment:
This gave me a pause, but I see what's happening here now. Might be worth adding a comment.

Also why doesn't `IDxcPixStructField` just subtype from `IDxcPixStructField0`?

Reply:
I didn't inherit from the old because JP's too-clever-for-me wrapper stuff can't handle it, and after half a day trying to make it do so, I gave up. One of these days I should just delete all that in favor of explicit error checking. It's hard to maintain and such a pain to step through.